ArduPlane/radio.cpp

```diff
@@ -280,7 +280,7 @@ void Plane::control_failsafe()
         // throttle has dropped below the mark
         failsafe.throttle_counter++;
         if (failsafe.throttle_counter == 10) {
-            gcs().send_text(MAV_SEVERITY_WARNING, "Throttle failsafe on");
+            gcs().send_text(MAV_SEVERITY_WARNING, "Throttle failsafe %s", "on");
```

Comment:
why do this? just wastes flash

Reply:
No it doesn't.  It saves a whole 12 bytes.  I did measure - but here, I'll do it again :-)

```
before: bin/arduplane    963308      2024   129140                965332           17696
 after: bin/arduplane    963296      2024   129136                965320           17712
```

common substring combination thingy...

Now I'm still kind of ambivalent about the patch, it certainly doesn't make the code easier to read.

Reply:
I notice that with the changes to failsafe_short_on_event() and failsafe_long_on_event(), the messages seem redundant. You get:
```
AP: Throttle failsafe on
AP: Failsafe. Short event ON
AP: Failsafe reason=1 (Radio Failsafe)
```
Could we not just remove the messages from radio.cpp?